### ISSUE TITLE
Audio streaming: Too buffering happy since version 2.1.0

### ISSUE THREAD

**Issue by falstaff1288** [CDrummond/cantata#1086]
Prior versions stopped instantaneously when MPD got paused.

Also, there's a long delay now when resuming from a paused session. Restarting the stream is necessary in order to liberate the audio buffer.

**Comment by CDrummond** [comment]
MPD **should** play silence on its HTTP output when paused. So, when you pause MPD cantata (as of 2.1.0) does **not** stop its QtMultimedia/VLC backend. But I've just noticed, when playback was resumed the backend was re-started - this *might* be the cause of your issue. I've modified git/master (for 2.2.0) to only start the backend if it is not already playing.

Can you checkout git/master and see if this resolves your issue?

**Comment by falstaff1288** [comment]
I've installed the git/master version and new there's no sound at all. Here's the error printed on the terminal:

[user@hostname ~]$ LANG=C cantata
Icon theme "cantata" not found.
GStreamer; Unable to pause - "http://192.168.1.200:8000"
Error: "No URI handler implemented for \"http\"."

**Comment by CDrummond** [comment]
Hmmm... I had changed the default back to using QtMultimedia for Linux builds. I have, jsut now, reverted back to VLC - as QtMultimedia seems to have issues when stream is disabled and re-enabled (no sound). The issue even persisted when re-starting Cantata.

Please update your checkout, and build again - making sure you have libvlc dev packages installed (so that VLC is used, and not QtMultimedia)

**Comment by falstaff1288** [comment]
Alright, I've compiled cantata-git (from the AUR/Archlinux); libVLC was disabled by default so I had to modify the PKGBUILD file.

- Now I'm getting audio but it seems like the streaming stays active even when MPD has been paused; therefore I'm getting very long delays after I resume MPD (like 20 seconds delay).

I prefer the older implementation which seemed, to my understanding, catch toggle/paused calls and ended the streaming. A new connection was imitated when MPD was toggled/played again. 

Again, I'm not sure if that's how it worked.
